Broken type check flow in the demos

![image](https://user-images.githubusercontent.com/4233843/81732255-eb30b880-9498-11ea-9a97-e43b095e3c61.png)

## 1.85.1 update

I am still getting errors with the the signatures.
Down to "only" 78 errors.

```
"C:\Program Files\nodejs\node.exe" C:\Users\shachar.soel@sap.com\AppData\Roaming\npm\node_modules\yarn\bin\yarn.js run compile
yarn run v1.22.5
$ tsc
../../node_modules/@sapui5/ts-types/types/sap.ca.ui.d.ts:5419:23 - error TS2304: Cannot find name 'FileSize'.

5419               oValue: FileSize
                           ~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.f.d.ts:10904:17 - error TS2304: Cannot find name 'integer'.

10904           row?: integer;
                      ~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.f.d.ts:10908:20 - error TS2304: Cannot find name 'integer'.

10908           column?: integer;
                         ~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.f.d.ts:10960:15 - error TS2304: Cannot find name 'integer'.

10960         iRow: integer,
                    ~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.f.d.ts:10964:18 - error TS2304: Cannot find name 'integer'.

10964         iColumn: integer
                       ~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.gantt.d.ts:4849:21 - error TS2430: Interface '$RelationshipSettings' incorrectly extends interface '$PathSettings'.
  Types of property 'selectedShape' are incompatible.
    Type 'SelectedRelationship' is missing the following properties from type 'SelectedShape': getHeight, setHeight

4849           interface $RelationshipSettings
                         ~~~~~~~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.gantt.d.ts:17630:34 - error TS2724: 'sap.gantt.simple' has no exported member named '$BaseDiamondSettings'. Did you
 mean '$AdhocDiamondSettings'?

17630         extends sap.gantt.simple.$BaseDiamondSettings {
                                       ~~~~~~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:25541:11 - error TS2559: Type 'Button' has no properties in common with type 'IFormContent'.

25541     class Button extends sap.ui.core.Control
                ~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:26863:11 - error TS2559: Type 'CheckBox' has no properties in common with type 'IFormContent'.

26863     class CheckBox extends sap.ui.core.Control
                ~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:40899:11 - error TS2559: Type 'Image' has no properties in common with type 'IFormContent'.

40899     class Image extends sap.ui.core.Control
                ~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:43560:11 - error TS2559: Type 'InputBase' has no properties in common with type 'IFormContent'.

43560     class InputBase extends sap.ui.core.Control
                ~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:45124:11 - error TS2559: Type 'Link' has no properties in common with type 'IFormContent'.

45124     class Link extends sap.ui.core.Control
                ~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:58978:11 - error TS2559: Type 'ObjectNumber' has no properties in common with type 'IFormContent'.

58978     class ObjectNumber extends sap.ui.core.Control
                ~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:59229:11 - error TS2559: Type 'ObjectStatus' has no properties in common with type 'IFormContent'.

59229     class ObjectStatus extends sap.ui.core.Control
                ~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:71422:11 - error TS2559: Type 'ProgressIndicator' has no properties in common with type 'IFormContent'.

71422     class ProgressIndicator extends sap.ui.core.Control
                ~~~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:73632:11 - error TS2559: Type 'RadioButton' has no properties in common with type 'IFormContent'.

73632     class RadioButton extends sap.ui.core.Control
                ~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:74128:11 - error TS2559: Type 'RadioButtonGroup' has no properties in common with type 'IFormContent'.

74128     class RadioButtonGroup extends sap.ui.core.Control
                ~~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:74713:11 - error TS2559: Type 'RatingIndicator' has no properties in common with type 'IFormContent'.

74713     class RatingIndicator extends sap.ui.core.Control
                ~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:76144:11 - error TS2420: Class 'ResponsiveScale' incorrectly implements interface 'IScale'.
  Property 'calcNumberOfTickmarks' is missing in type 'ResponsiveScale' but required in type 'IScale'.

76144     class ResponsiveScale extends sap.ui.core.Element implements sap.m.IScale {
                ~~~~~~~~~~~~~~~

  ../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:9664:7
    9664       calcNumberOfTickmarks(
               ~~~~~~~~~~~~~~~~~~~~~~
    9665         /**
         ~~~~~~~~~~~
     ...
    9668         mOptions: object
         ~~~~~~~~~~~~~~~~~~~~~~~~
    9669       ): number;
         ~~~~~~~~~~~~~~~~
    'calcNumberOfTickmarks' is declared here.

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:76525:11 - error TS2559: Type 'SearchField' has no properties in common with type 'IFormContent'.

76525     class SearchField extends sap.ui.core.Control
                ~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:78153:11 - error TS2559: Type 'Select' has no properties in common with type 'IFormContent'.

78153     class Select extends sap.ui.core.Control
                ~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:83907:11 - error TS2559: Type 'Slider' has no properties in common with type 'IFormContent'.

83907     class Slider extends sap.ui.core.Control
                ~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:87806:11 - error TS2559: Type 'StepInput' has no properties in common with type 'IFormContent'.

87806     class StepInput extends sap.ui.core.Control
                ~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:88583:11 - error TS2559: Type 'Switch' has no properties in common with type 'IFormContent'.

88583     class Switch extends sap.ui.core.Control
                ~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.m.d.ts:92090:11 - error TS2559: Type 'Text' has no properties in common with type 'IFormContent'.

92090     class Text extends sap.ui.core.Control
                ~~~~

../../node_modules/@sapui5/ts-types/types/sap.suite.ui.commons.d.ts:23890:15 - error TS2559: Type 'MicroProcessFlow' has no properties in common with type 'IForm
Content'.

23890         class MicroProcessFlow extends sap.ui.core.Control
                    ~~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.tnt.d.ts:249:11 - error TS2559: Type 'InfoLabel' has no properties in common with type 'IFormContent'.

249     class InfoLabel extends sap.ui.core.Control
              ~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.commons.d.ts:7128:13 - error TS2559: Type 'Button' has no properties in common with type 'IFormContent'.

7128       class Button extends sap.ui.core.Control
                 ~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.commons.d.ts:8426:13 - error TS2559: Type 'CheckBox' has no properties in common with type 'IFormContent'.

8426       class CheckBox extends sap.ui.core.Control
                 ~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.commons.d.ts:10789:13 - error TS2559: Type 'Image' has no properties in common with type 'IFormContent'.

10789       class Image extends sap.ui.core.Control
                  ~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.commons.d.ts:11887:13 - error TS2559: Type 'Link' has no properties in common with type 'IFormContent'.

11887       class Link extends sap.ui.core.Control
                  ~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.commons.d.ts:14944:13 - error TS2559: Type 'RadioButton' has no properties in common with type 'IFormContent'.

14944       class RadioButton extends sap.ui.core.Control
                  ~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.commons.d.ts:15322:13 - error TS2559: Type 'RadioButtonGroup' has no properties in common with type 'IFormConten
t'.

15322       class RadioButtonGroup extends sap.ui.core.Control
                  ~~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.commons.d.ts:19022:13 - error TS2559: Type 'SegmentedButton' has no properties in common with type 'IFormContent
'.

19022       class SegmentedButton extends sap.ui.core.Control
                  ~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.commons.d.ts:19247:13 - error TS2559: Type 'Slider' has no properties in common with type 'IFormContent'.

19247       class Slider extends sap.ui.core.Control
                  ~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.commons.d.ts:20858:13 - error TS2559: Type 'TextField' has no properties in common with type 'IFormContent'.

20858       class TextField extends sap.ui.core.Control
                  ~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.commons.d.ts:21548:13 - error TS2559: Type 'TextView' has no properties in common with type 'IFormContent'.

21548       class TextView extends sap.ui.core.Control
                  ~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.comp.d.ts:10140:15 - error TS2559: Type 'SmartField' has no properties in common with type 'IFormContent'.

10140         class SmartField extends sap.ui.core.Control
                    ~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.comp.d.ts:25169:46 - error TS2339: Property 'fl' does not exist on type 'typeof ui'.

25169         class TransportDialog extends sap.ui.fl.transport.TransportDialog {
                                                   ~~

../../node_modules/@sapui5/ts-types/types/sap.ui.core.d.ts:2898:13 - error TS2420: Class 'Event' incorrectly implements interface 'Poolable'.
  Type 'Event' is missing the following properties from type 'Poolable': init, reset

2898       class Event extends sap.ui.base.Object implements sap.ui.base.Poolable {
                 ~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.core.d.ts:11776:15 - error TS2417: Class static side 'typeof XMLView' incorrectly extends base class static side
 'typeof View'.
  Types of property 'registerPreprocessor' are incompatible.
    Type '(sType: any, vPreprocessor: TimerHandler, bSyncSupport: boolean, bOnDemand?: boolean, mSettings?: object) => void' is not assignable to type '(sType: s
tring, vPreprocessor: TimerHandler, sViewType: string, bSyncSupport: boolean, bOnDemand?: boolean, mSettings?: object) => void'.
      Types of parameters 'bSyncSupport' and 'sViewType' are incompatible.
        Type 'string' is not assignable to type 'boolean'.

11776         class XMLView extends sap.ui.core.mvc.View {
                    ~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.core.d.ts:24777:13 - error TS2559: Type 'Icon' has no properties in common with type 'IFormContent'.

24777       class Icon extends sap.ui.core.Control
                  ~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.generic.app.d.ts:1780:14 - error TS2339: Property 'BaseController' does not exist on type 'typeof transaction'.

1780             .BaseController {
                  ~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.generic.app.d.ts:2001:14 - error TS2339: Property 'BaseController' does not exist on type 'typeof transaction'.

2001             .BaseController {
                  ~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.generic.app.d.ts:2122:12 - error TS2339: Property 'BaseController' does not exist on type 'typeof transaction'.

2122           .BaseController {
                ~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.layout.d.ts:238:15 - error TS2420: Class 'CSSGrid' incorrectly implements interface 'IGridConfigurable'.
  Type 'CSSGrid' is missing the following properties from type 'IGridConfigurable': getGridDomRefs, getGridLayoutConfiguration

238         class CSSGrid extends sap.ui.core.Control
                  ~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.layout.d.ts:1082:44 - error TS2724: 'sap.ui.layout.cssgrid' has no exported member named 'IGridItemLayoutData'.
Did you mean 'GridItemLayoutData'?

1082           implements sap.ui.layout.cssgrid.IGridItemLayoutData {
                                                ~~~~~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.layout.d.ts:1926:44 - error TS2724: 'sap.ui.layout.cssgrid' has no exported member named 'IGridItemLayoutData'.
Did you mean 'GridItemLayoutData'?

1926           implements sap.ui.layout.cssgrid.IGridItemLayoutData {
                                                ~~~~~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.unified.d.ts:11321:13 - error TS2559: Type 'FileUploader' has no properties in common with type 'IFormContent'.

11321       class FileUploader extends sap.ui.core.Control
                  ~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:68:15 - error TS2420: Class 'BaseNodeProxy' incorrectly implements interface 'Poolable'.
  Type 'BaseNodeProxy' is missing the following properties from type 'Poolable': init, reset

68         class BaseNodeProxy extends sap.ui.vk.BaseNodeProxy
                 ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:2067:15 - error TS2420: Class 'BaseNodeProxy' incorrectly implements interface 'Poolable'.
  Type 'BaseNodeProxy' is missing the following properties from type 'Poolable': init, reset

2067         class BaseNodeProxy extends sap.ui.vk.BaseNodeProxy
                   ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:3008:15 - error TS2420: Class 'BaseNodeProxy' incorrectly implements interface 'Poolable'.
  Type 'BaseNodeProxy' is missing the following properties from type 'Poolable': init, reset

3008         class BaseNodeProxy extends sap.ui.vk.BaseNodeProxy
                   ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:4801:35 - error TS2694: Namespace 'sap.ui.vk.tools' has no exported member '$ToolSettings'.

4801           extends sap.ui.vk.tools.$ToolSettings {
                                       ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:4854:35 - error TS2694: Namespace 'sap.ui.vk.tools' has no exported member '$ToolSettings'.

4854           extends sap.ui.vk.tools.$ToolSettings {
                                       ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:4908:35 - error TS2694: Namespace 'sap.ui.vk.tools' has no exported member '$ToolSettings'.

4908           extends sap.ui.vk.tools.$ToolSettings {
                                       ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:4917:64 - error TS2694: Namespace 'sap.ui.vk.tools' has no exported member '$ToolSettings'.

4917         interface $HitTestToolSettings extends sap.ui.vk.tools.$ToolSettings {
                                                                    ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:4929:61 - error TS2694: Namespace 'sap.ui.vk.tools' has no exported member '$ToolSettings'.

4929         interface $MoveToolSettings extends sap.ui.vk.tools.$ToolSettings {
                                                                 ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:5001:35 - error TS2694: Namespace 'sap.ui.vk.tools' has no exported member '$ToolSettings'.

5001           extends sap.ui.vk.tools.$ToolSettings {
                                       ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:5009:64 - error TS2694: Namespace 'sap.ui.vk.tools' has no exported member '$ToolSettings'.

5009         interface $RedlineToolSettings extends sap.ui.vk.tools.$ToolSettings {
                                                                    ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:5021:35 - error TS2694: Namespace 'sap.ui.vk.tools' has no exported member '$ToolSettings'.

5021           extends sap.ui.vk.tools.$ToolSettings {
                                       ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:5028:63 - error TS2694: Namespace 'sap.ui.vk.tools' has no exported member '$ToolSettings'.

5028         interface $RotateToolSettings extends sap.ui.vk.tools.$ToolSettings {
                                                                   ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:5109:35 - error TS2694: Namespace 'sap.ui.vk.tools' has no exported member '$ToolSettings'.

5109           extends sap.ui.vk.tools.$ToolSettings {
                                       ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:5116:62 - error TS2694: Namespace 'sap.ui.vk.tools' has no exported member '$ToolSettings'.

5116         interface $ScaleToolSettings extends sap.ui.vk.tools.$ToolSettings {
                                                                  ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:5194:35 - error TS2694: Namespace 'sap.ui.vk.tools' has no exported member '$ToolSettings'.

5194           extends sap.ui.vk.tools.$ToolSettings {
                                       ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:5200:64 - error TS2694: Namespace 'sap.ui.vk.tools' has no exported member '$ToolSettings'.

5200         interface $TooltipToolSettings extends sap.ui.vk.tools.$ToolSettings {
                                                                    ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:10702:49 - error TS2339: Property 'Element' does not exist on type 'typeof base'.

10702       class AnimationPlayer extends sap.ui.base.Element {
                                                      ~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:11586:13 - error TS2420: Class 'BaseNodeProxy' incorrectly implements interface 'Poolable'.
  Type 'BaseNodeProxy' is missing the following properties from type 'Poolable': init, reset

11586       class BaseNodeProxy extends sap.ui.base.Object
                  ~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:22212:30 - error TS2694: Namespace 'sap.ui.vk' has no exported member 'IPlaybackCollection'.

22212         implements sap.ui.vk.IPlaybackCollection {
                                   ~~~~~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vk.d.ts:23586:13 - error TS2371: A parameter initializer is only allowed in a function or constructor implementa
tion.

23586             oWebGLContextAttributes = {
                  ~~~~~~~~~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ui.vtm.d.ts:2052:17 - error TS2430: Interface '$ColumnSettings' incorrectly extends interface '$ElementSettings'.
  Types of property 'tooltip' are incompatible.
    Type 'string' is not assignable to type 'TooltipBase'.

2052       interface $ColumnSettings extends sap.ui.core.$ElementSettings {
                     ~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.uiext.inbox.d.ts:2998:44 - error TS2694: Namespace 'sap.uiext.inbox' has no exported member 'InboxPrimaryFilterEnum
'.

2998         setFilter(oFilter: sap.uiext.inbox.InboxPrimaryFilterEnum): void;
                                                ~~~~~~~~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ushell.d.ts:21:18 - error TS2451: Cannot redeclare block-scoped variable 'Container'.

21     export const Container: sap.ushell.services.Container;
                    ~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ushell.d.ts:1637:33 - error TS2304: Cannot find name 'ContentNode'.

1637           vContainer?: object | ContentNode | ContentNode[]
                                     ~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ushell.d.ts:1637:47 - error TS2304: Cannot find name 'ContentNode'.

1637           vContainer?: object | ContentNode | ContentNode[]
                                                   ~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ushell.d.ts:5356:15 - error TS2451: Cannot redeclare block-scoped variable 'Container'.

5356     namespace Container {
                   ~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ushell.d.ts:8220:29 - error TS2694: Namespace 'sap.ui.core' has no exported member '$XMLCompositeSettings'.

8220         extends sap.ui.core.$XMLCompositeSettings {
                                 ~~~~~~~~~~~~~~~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.ushell.d.ts:8400:48 - error TS2339: Property 'service' does not exist on type 'typeof core'.

8400       class ShellUIService extends sap.ui.core.service.Service {
                                                    ~~~~~~~

../../node_modules/@sapui5/ts-types/types/sap.viz.d.ts:36:36 - error TS2694: Namespace 'sap.viz.ui5.api.env' has no exported member 'Format'.

36             ): sap.viz.ui5.api.env.Format;
                                      ~~~~~~


Found 78 errors.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

--- a/demos/demo-js/package.json
+++ b/demos/demo-js/package.json
@@ -5,10 +5,10 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@sapui5/ts-types": "1.85.1",
-    "typescript": "4.1.2"
+    "typescript": "4.1.3"
   },
   "scripts": {
-    "test": "npm run type-check",
-    "type-check": "echo \"tsc\""
+    "ci": "npm-run-all type-check",
+    "type-check": "tsc"
   }
 }

--- a/demos/demo-js/tsconfig.json
+++ b/demos/demo-js/tsconfig.json
@@ -4,6 +4,7 @@
     "noEmit": true,
     "checkJs": true,
     "allowJs": true,
+    "skipLibCheck": true,
     "types": ["@sapui5/ts-types"]
   }
 }

--- a/demos/demo-ts/package.json
+++ b/demos/demo-ts/package.json
@@ -5,9 +5,10 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@sapui5/ts-types": "1.85.1",
-    "typescript": "4.1.2"
+    "typescript": "4.1.3"
   },
   "scripts": {
-    "compile": "echo \"tsc\""
+    "ci": "npm-run-all compile",
+    "compile": "tsc"
   }
 }

--- a/demos/demo-ts/tsconfig.json
+++ b/demos/demo-ts/tsconfig.json
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "module": "none",
     "types": ["@sapui5/ts-types"],
-    "outDir": "lib"
+    "outDir": "lib",
+    "skipLibCheck": true
   }
 }

